pull_request_package/Gemfile

```diff
@@ -4,6 +4,5 @@ source "https://rubygems.org"
 
 git_source(:github) {|repo_name| "https://github.com/#{repo_name}" }
 
-gem 'octokit'
-gem 'nokogiri'
-gem 'activemodel'
+gem 'slop'
+gem 'pull_request_builder', github: 'vpereira/pull_request_builder'
```

Comment:
Hm shouldn't the gem's source be here also?

Reply:
I wasn't thinking on pack this as gem, just the ```pull_request_builder``` but if you want, I can do it

Reply:
Why not?

Reply:
I don't think this way you can build the container on OBS